docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -164,3 +164,15 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GB of memory.
 ====
 
+== Eviction Threshold
+
+A node running applications requires enough reserve memomory. An application can trigger an out-of-memory event when it uses more than its reserved memory.
```

Comment:
This is a basic typo, you will note that Vale is mandatory and yet you are still not using it.